File: ProductosServicios/src/Productos/Infrestructure/DTOS/Responses/BaseResponse.py
class BaseResponse:
    def __init__(self, data, message, status, httpstatuscode):
        self.data = data
        self.message = message
        self.status = status
        self.httpstatuscode = httpstatuscode

    def response(self):
        return {
            "data": self.data,
            "message": self.message,
            "status": self.status
        }, self.httpstatuscode

File: OrdenesService/src/main/java/com/example/ordenesservice/Ordenes/Infrestructure/dtos/responses/BaseResponse.java
package com.example.ordenesservice.Ordenes.Infrestructure.dtos.responses;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;
import org.springframework.http.HttpStatus;

@Getter @Builder
public class BaseResponse {

    private Object data;
    private String message;
    private Boolean success;
    private HttpStatus httpStatus;

}
